### ISSUE TITLE
Academic Writer (Professional Version) is not found.

### ISSUE THREAD

**Issue by GitKalina** [ai-boost/Awesome-GPTs#280]
https://chat.openai.com/g/g-Ej5zYQRIB-academic-writer-professional-version doesn't work.

**Comment by joaocarlos** [comment]
As such, several other awesome GPTs have gone. Does anyone have an explanation for what is happening?

**Comment by impressivewebs** [comment]
Yeah, every single link I clicked is a 404 not found. Not sure what's going on here.

**Comment by ctptripz** [comment]
I thing it moved to a different  gpt

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
